pageserver/src/tenant/storage_layer/layer.rs

```diff
@@ -1034,11 +1039,13 @@ impl LayerInner {
                 Ok(())
             }
             Err(e) if e.kind() == std::io::ErrorKind::NotFound => {
-                tracing::info!("failed to evict file from disk, it was already gone");
+                tracing::error!(
+                    "failed to evict file from disk, it was already gone (metrics will be off)"
```

Comment:
"off" as in "turned off" or as in "off by some amount"? Maybe it would be clearer to express it differently?

Reply:
I meant "off as in wrong", I don't see how a metric could be disabled. I don't know how to express this better.

It might be even correct to still substract the value from the counter.

Reply:
```suggestion
                    "failed to evict file from disk, it was already gone (metrics will become wrong)"
```

Reply:
inaccurate used in: [94d6a72](https://github.com/neondatabase/neon/pull/5766/commits/94d6a72eff134ca3ffeeea727cbf31a5d26d417a)